src/test/java/openfasttrack/importer/markdown/MarkdownTestConstants.java

```diff
@@ -0,0 +1,66 @@
+package openfasttrack.importer.markdown;
+
+/*-
+ * #%L
+ * OpenFastTrack
+ * %%
+ * Copyright (C) 2016 - 2018 hamstercommunity
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
+import openfasttrack.core.SpecificationItemId;
+
+public class MarkdownTestConstants
+{
+    private MarkdownTestConstants()
+    {
+        // not instantiable
+    }
+
+    final static SpecificationItemId ID1 = SpecificationItemId.parseId("type~id~1");
+    final static SpecificationItemId ID2 = SpecificationItemId.parseId("type~id~2");
+    final static String TITLE = "Requirement Title";
```

Comment:
Inconsistent order of modifiers. I think `static final` is standard.

Reply:
Fixed.